ironfish-cli/src/commands/wallet/import.ts

```diff
@@ -19,14 +19,19 @@ export class ImportCommand extends IronfishCommand {
       default: true,
       description: 'Rescan the blockchain once the account is imported',
     }),
```

Comment:
This no longer matches the signature of the `export` command. I think we should make the `path` a labeled argument on the other command as well (instead of a positional argument). 

Also I would name this `path` not `importPath`. I am guessing there is a `--importPath` flag currently.

Last do you think `p` is a good char choice? Honest question, I was checking around saw one or two other commands use that flag as path, but also saw some using it as port

Reply:
Good ideas, implementing. As far as `p` goes, I personally prefer just using the long argument, so I am inclined to remove the `char` entirely to make it unambiguous.